### PR TITLE
adjust(functions & objects): add variable in object structure

### PR DESCRIPTION
**Pull request by MuneesDeepan28**

### CODE REVIEW

**Comment by MuneesDeepan28** [comment]
https://github.com/MuneesDeepan28/JSTraining/issues/8

File: functions/functions.html [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+				'subject5': 43
+			},
+		};
+		var jsTraining = jsTraining || {
+		};
+		jsTraining.studentMarkSystem = {
+			"printMarkDetails": function () {
+				var total = 0;
+				for (let markList in studentDetails) {
+					console.log(studentDetails[markList]);
```
Comment (pradeep3093) [inline functions/functions.html]: is this correct? you are printing data inside the loop..@ @MuneesDeepan28

File: functions/functions.html [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+					total = studentDetails[markList].subject1 + studentDetails[markList].subject2 + studentDetails[markList].subject3 + studentDetails[markList].subject4 + studentDetails[markList].subject5;
+					studentDetails[markList].Total = total;
+				}
+			},
+			"printMarkDetailsAsc": function () {
+				var sortStudentDetails = Object.values(studentDetails);
```
Comment (pradeep3093) [inline functions/functions.html]: why var needed here?

File: functions/functions.html [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+				var sortAscending = sortStudentDetails.sort(function (a, b) {
+					return a.Total - b.Total;
+				});
+				console.table(sortAscending);
+			},
+			"printMarkDetailsDesc": function () {
+				var sortStudentDetails = Object.values(studentDetails);
+				var sortDescending = sortStudentDetails.sort(function (a, b) {
```
Comment (pradeep3093) [inline functions/functions.html]: why where needed here

Comment (pradeep3093) [reply functions/functions.html]: variable naming also wrong.. you are fetching values from object then how it could be sortStudentDetails?

**Pull request merged**